task/mgmt-gateway/src/update/sp.rs

```diff
@@ -11,19 +12,54 @@ use gateway_messages::{
     UpdateInProgressStatus, UpdateStatus,
 };
 
+#[cfg_attr(not(feature = "auxflash"), path = "sp/stub_auxflash.rs")]
+mod auxflash;
```

Comment:
This is a little confusing – maybe move it into the `cfg_if!` block, e.g.
```rust
cfg_if! {
    if #[cfg(feature = "auxflash")] {
        mod auxflash;
    } else {
        mod stub_auxflash;
        use stub_auxflash as auxflash;
   }
```
(if that's allowed?)

Reply:
It is! Changed in 5c631443